openapi.yaml

```diff
@@ -4820,6 +4855,37 @@ components:
                 spaced steps.
               type: string
               nullable: true
+    geometry_type:
+      title: Geometry type
+      type: string
+      enum:
+        - Point
+        - MultiPoint
+        - LineString
+        - MultiLineString
+        - Polygon
+        - MultiPolygon
+        - GeometryCollection
```

Comment:
Shouldn't there be  a note here (like in other places) that GeometryCollection should be avoided?

Reply:
No, I don't think it makes a lot of sense here. This is backend-oriented documentation. Back-ends can't really "avoid" GeometryCollections, they either support it or not. The data that may consist of GeometryCollections is provided in a different place (processes) by a different audience (end-users) and for this case it's documented in the processes.